libraries/AP_Beacon/AP_Beacon_Backend.cpp

```diff
@@ -68,6 +68,9 @@ void AP_Beacon_Backend::set_beacon_position(uint8_t beacon_instance, const Vecto
 
     // set position after correcting yaw
     _frontend.beacon_state[beacon_instance].position = correct_for_orient_yaw(pos);
+
+    // update boundary for fence
+    _frontend.update_boundary_points();
```

Comment:
Why do this here? Wouldn't it be better to do it in the update method of the frontend, after doing the update of the backend?